Spatial.gd

```diff
@@ -52,7 +52,10 @@ func _ready():
 	x11_fg = X11_FG.new()
 	x11_window_id_burrito = OS.get_native_handle(OS.WINDOW_HANDLE)
 	OS.window_maximized = false
-	OS.window_size = Vector2(1920, 1080)
+	var size_tuple = x11_fg.get_gw2_geometry()
+	OS.window_size = Vector2(size_tuple[0], size_tuple[1])
+	# Postion window in center of screen
+	OS.set_window_position(Vector2(0,0))
```

Comment:
Note that I still set the window to begin at the top left, vs the original behavior. It will need to be dragged anyway, and at least this way, if the user is using the full screen, he doesn't need to reposition it.